docs/user-documentation/content_types.md

```diff
@@ -133,36 +133,36 @@ The mapping itself consists of the `types`' and `fieldMappings` configurations.
 
 Namespaces currently supported (ordered by the module that supplies them) include:
- * rdf
-      * content: http://purl.org/rss/1.0/modules/content/
-      * dc: http://purl.org/dc/terms/
-      * foaf: http://xmlns.com/foaf/0.1/
-      * og: http://ogp.me/ns#
-      * rdfs: http://www.w3.org/2000/01/rdf-schema#
-      * schema: http://schema.org/
-      * sioc: http://rdfs.org/sioc/ns#
-      * sioct: http://rdfs.org/sioc/types#
-      * skos: http://www.w3.org/2004/02/skos/core#
-      * xsd: http://www.w3.org/2001/XMLSchema#
- * islandora
-      * ldp: http://www.w3.org/ns/ldp#
-      * dc11: http://purl.org/dc/elements/1.1/
-      * nfo: http://www.semanticdesktop.org/ontologies/2007/03/22/nfo/v1.1/
-      * ebucore: http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#
-      * fedora: http://fedora.info/definitions/v4/repository#
-      * owl: http://www.w3.org/2002/07/owl#
-      * ore: http://www.openarchives.org/ore/terms/
-      * rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
-      * islandora: http://islandora.ca/CLAW/
-      * pcdm: http://pcdm.org/models#
-      * use: http://pcdm.org/use#
-      * iana: http://www.iana.org/assignments/relation/
- * islandora_demo
-      * relators: http://id.loc.gov/vocabulary/relators/
- * controlled_access_terms
-      * wgs84_pos: http://www.w3.org/2003/01/geo/wgs84_pos#
-      * org: https://www.w3.org/TR/vocab-org/#org:
-      * xs: http://www.w3.org/2001/XMLSchema#
+ - rdf
+      - content: http://purl.org/rss/1.0/modules/content/
+      - dc: http://purl.org/dc/terms/
+      - foaf: http://xmlns.com/foaf/0.1/
+      - og: http://ogp.me/ns#
+      - rdfs: http://www.w3.org/2000/01/rdf-schema#
+      - schema: http://schema.org/
+      - sioc: http://rdfs.org/sioc/ns#
+      - sioct: http://rdfs.org/sioc/types#
+      - skos: http://www.w3.org/2004/02/skos/core#
+      - xsd: http://www.w3.org/2001/XMLSchema#
+ - islandora
+      - ldp: http://www.w3.org/ns/ldp#
+      - dc11: http://purl.org/dc/elements/1.1/
+      - nfo: http://www.semanticdesktop.org/ontologies/2007/03/22/nfo/v1.1/
+      - ebucore: http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#
+      - fedora: http://fedora.info/definitions/v4/repository#
+      - owl: http://www.w3.org/2002/07/owl#
+      - ore: http://www.openarchives.org/ore/terms/
+      - rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+      - islandora: http://islandora.ca/CLAW/
+      - pcdm: http://pcdm.org/models#
+      - use: http://pcdm.org/use#
+      - iana: http://www.iana.org/assignments/relation/
+ - islandora_demo
+      - relators: http://id.loc.gov/vocabulary/relators/
+ - controlled_access_terms
+      - wgs84_pos: http://www.w3.org/2003/01/geo/wgs84_pos#
+      - org: https://www.w3.org/TR/vocab-org/#org:
+      - xs: http://www.w3.org/2001/XMLSchema#
```

Comment:
This is not working, I think the hyphens are correct (as the below changes in Further Reading look correct) but here its a huge block of text. I'll play with it, but also you can install mkdocs on your machine and then in the CLAW directory run `mkdocs serve` which will build your docs and serve them on `localhost:8111` to check them

Reply:
I'm sure this is my fault. There shouldn't be a leading space before the first-level items' hyphen and, [according to mkdocs](https://python-markdown.github.io/#differences), there should _only_ be 4 spaces before the sub-items, _not_ 6. 
```
- controlled_access_terms
    - wgs84_pos: http://www.w3.org/2003/01/geo/wgs84_pos#
    - org: https://www.w3.org/TR/vocab-org/#org:
    - xs: http://www.w3.org/2001/XMLSchema#
```

Reply:
Updated the PR accordingly